packages/ketcher-polymer-editor-react/src/components/monomerLibrary/monomerLibraryList/MonomerList.tsx

```diff
@@ -41,15 +42,33 @@ const MonomerList = ({ onItemClick, libraryName }: IMonomerListProps) => {
 
   const groups = selectMonomerGroups(items);
 
+  const [selectedMonomers, setSelectedMonomers] = useState('');
+
+  const selectItem = (monomer: MonomerItemType) => {
+    setSelectedMonomers(monomer.label);
+  };
+
   return (
     <MonomerListContainer>
-      {groups.map(({ groupItems, groupTitle }) => {
+      {groups.map(({ groupItems, groupTitle }, index) => {
+        if (groupTitle === '.') {
+          return (
+            <MonomerGroup
+              key={index}
+              items={groupItems}
+              onItemClick={onItemClick || selectItem}
+              selectedMonomerLabel={selectedMonomers}
+            />
+          );
+        }
+
         return (
           <MonomerGroup
             key={groupTitle}
             title={groupTitle}
             items={groupItems}
-            onItemClick={onItemClick}
+            onItemClick={onItemClick || selectItem}
+            selectedMonomerLabel={selectedMonomers}
           />
         );
       })}
```

Comment:
I believe that we can pass title only if there are more that one groups. In this case code would be:
```
 return (
          <MonomerGroup
            key={groupTitle}
            title={groups.length === 1 ? null : groupTitle}
            items={groupItems}
            onItemClick={onItemClick}
            onItemClick={onItemClick || selectItem}
            selectedMonomerLabel={selectedMonomers}
          />
        );
```